plugins/jobs/src/js/__tests__/JobsList-test.ts

```diff
@@ -0,0 +1,3 @@
+describe("JobsList", () => {
+  it("tests");
```

Comment:
todo: not sure if I need to add some tests here 🤔 theoretically everything should be there…